src/use_case/create_meeting/CreateMeetingDataAccessInterface.java

```diff
@@ -1,13 +1,14 @@
 package use_case.create_meeting;
 
 import java.sql.Time;
+import java.util.ArrayList;
+import entity.Meeting;
 
-public class CreateMeetingDataAccessInterface {
-    public CreateMeetingDataAccessInterface() {
+public interface CreateMeetingDataAccessInterface {
         void saveMeeting(String projectName, Meeting newMeeting);
 
         boolean projectNameExists(String projectName, String meetingName);
 
-        boolean memberExists(String projectName, ArrayList<String> ParticipantEmail);
-    }
+        boolean memberExists(String projectName, ArrayList<String> participantEmail);
```

Comment:
suggestion: since we are going to have method for memberExists() with the second parameter as single participant email, you can reuse that by making a for loop (so we don't have to add new methods for doing the same thing)